third_party/move/move-bytecode-verifier/src/check_duplication.rs

```diff
@@ -201,6 +205,50 @@ impl<'a> DuplicationChecker<'a> {
         Ok(())
     }
 
+    fn check_variant_field_handles(&self) -> PartialVMResult<()> {
```

Comment:
Can we have tests for these?

Reply:
There is an open bug for creating a better test framework here. Its very difficult and not well maintainable to add tests for 'invalid' bytecode, because the compiler does only produce valid ones. A lot of features like this are untested today. However, this is just boilerplate of existing code, so I'm not too concerned in this case.